openshiftcontrolplane/v1/types.go

```diff
@@ -46,8 +46,38 @@ type OpenShiftAPIServerConfig struct {
 
 	// TODO this needs to be removed.
 	APIServerArguments map[string][]string `json:"apiServerArguments"`
+
+	// apiServers holds information about enabled/disabled API servers
+	APIServers APIServers `json:"apiServers"`
+}
+
+type APIServers struct {
```

Comment:
to work more cleanly with server-side-apply to avoid conflicts (yeah, this is a file-only API, but good practice here), you want to have list items keyed by something unique.  so you want something like

```
type PerGroupOptions struct{
  Name string

  // EnabledVersions is a list of versions that must be enabled in addition to the defaults
  EnabledVersions string
  
  // DisabledVersions is a list of versions that must be disabled in addition to the defaults
  DisabledVersions string
}
```